package.json

```diff
@@ -254,6 +254,7 @@
     "@chromatic-com/storybook": "1.2.23",
     "@leather-wallet/prettier-config": "0.0.5",
     "@leather-wallet/rpc": "0.3.0",
+    "@leather-wallet/panda-preset": "link:../mono/packages/panda-preset/",
```

Comment:
I am using `link` here as I couldn't get `file` working, maybe I am missing something

Reply:
Does `pnpm link --global .` in preset & `pnpm link @leather-wallet/panda-preset` in ext work?

Reply:
I gave that a try and it didn't work immediately. It could be possible to get it to work with some more investigation. 

**panda-preset**
At first `pnpm link --global .` failed right away with:

```
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In ../../../../../Library/pnpm/global/5: "@leather-wallet/tokens@workspace:*" is in the dependencies but no package named "@leather-wallet/tokens" is present in the workspace
```

I changed the import to `"@leather-wallet/tokens": "0.4.0"` and it then succeeded although with a different warning ` ERR_PNPM_PATCH_NOT_APPLIED  The following patches were not applied: zustand@4.5.2` 

That's not something we want to do though as I think packages should refer to the workspace version `workspace:*` 


**extension**
In the extension `pnpm link @leather-wallet/panda-preset` fails with:
```
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/Users/petew/src/leather/extension/node_modules/@leather-wallet/panda-preset".
 ```
 
 I can investigate more. It could be that I also need to add some `pnpm` overrides. I was following [Edgar's guide](https://github.com/leather-wallet/mono/blob/dev/docs/extension-development.md) but got it working just changing from the `file:` protocol to `link:`